src/main/webapp/WEB-INF/jsp/access_confirmation.jsp

```diff
@@ -30,28 +32,33 @@
 	<c:remove scope="session" var="SPRING_SECURITY_LAST_EXCEPTION" />
 
 	<authz:authorize ifAllGranted="ROLE_OPENMRS">
-		<h2>Please Confirm</h2>
+		<h1 styleClass="h3 mb-3 font-weight-normal">Please Confirm</h1>
 
 		<p>
 			You hereby authorize "
 			<c:out value="${client.clientId}" />
 			" to access your protected resources.
 		</p>
-
-		<form id="confirmationForm" name="confirmationForm"
-			action="<%=request.getContextPath()%>/oauth/authorize" method="post">
-			<input name="user_oauth_approval" value="true" type="hidden" /> <label>
-				<input name="authorize" value="Authorize" type="submit" />
-			</label>
+		<br>
+		<form id="confirmationForm" name="confirmationForm" action="<%=request.getContextPath()%>/oauth/authorize"
+			method="post">
+			<div styleClass="form-group">
+				<input name="user_oauth_approval" value="true" type="hidden" /> <label>
+					<input name="authorize" styleClass="btn btn-primary btn-lg btn-block" type="submit" value="Authorize" />
```

Comment:
I'd prefer that you don't use `btn-lg` so that the buttons are normal-sized.  But @rowo has the last word on this

Reply:
I agree